actionview/lib/action_view/helpers/tags/collection_select.rb

```diff
@@ -14,16 +14,24 @@ def initialize(object_name, method_name, template_object, collection, value_meth
         end
 
         def render
-          option_tags_options = {
-            selected: @options.fetch(:selected) { value },
-            disabled: @options[:disabled]
-          }
```

Comment:
These lines are currently (almost) identical to their counterparts in the `Select` tag helper class:

https://github.com/rails/rails/blob/436207aa2011463e49feb6310fa144bb0a96bf0c/actionview/lib/action_view/helpers/tags/select.rb#L17-L20

I don't think we should refactor them, and hide the equivalence, without a good reason.

Reply:
Yes, I noticed the same pattern in [`GroupedCollectionSelect`](https://github.com/rails/rails/blob/main/actionview/lib/action_view/helpers/tags/grouped_collection_select.rb#L19-L22), too.

In #44335, I factored this out into a [`SelectRenderer`](https://github.com/rails/rails/pull/44335/files#diff-cfd49011c832807e86c2316a2b44caa519d0dbcc3fb2ef3e72edfb68c80406e0) module.  (So `Select` would look like [this](https://github.com/rails/rails/pull/44335/files#diff-5937cc0ef86a3454d12606314ef6159428001e076674545c40ac40ddd8c67860L16-R35), and `CollectionSelect` would [match](https://github.com/rails/rails/pull/44335/files#diff-dda2312e5ec8602aadaf50abed53ce2f873734495077331327d70f1d36a4baeeL16-R23).)  I was planning to merge that PR first, and then rebase this PR on top of it.  Any objections?